iron-autogrow-textarea.html

```diff
@@ -108,6 +105,7 @@
     <!-- size the input/textarea with a div, because the textarea has intrinsic size in ff -->
     <div class="textarea-container fit">
       <textarea id="textarea"
+        name$="[[name]]"
```

Comment:
Yeah, when we remove `bindValue`, that's totally the way to go, but I don't think it would help right now (since we still have to maintain the horrid `bindValue` path) 😭